src/io/pithos/blob.clj

```diff
@@ -52,6 +52,10 @@
   set a large limit of chunks to retrieve from a block."
   524288)
 
+(def slice-width
+  "Provide a sensible default for fetching chunk slices."
+  1024)
```

Comment:
how did you come up with this number (compared to the previous 8k in absolute-chunk-limit)?

is absolute-chunk-limit still needed?